exe/wallet/Main.hs

```diff
@@ -173,20 +182,34 @@ exec manager args
             (ApiT wPwd)
 
     | args `isPresent` command "wallet" && args `isPresent` command "update" = do
-        wId <- args `parseArg` longOption "id"
+        wId <- args `parseArg` longOption "wallet-id"
         wName <- args `parseArg` longOption "name"
         runClient $ putWallet (ApiT wId) $ WalletPutData
             (Just $ ApiT wName)
 
     | args `isPresent` command "wallet" && args `isPresent` command "delete" = do
-        wId <- args `parseArg` longOption "id"
+        wId <- args `parseArg` longOption "wallet-id"
         runClient $ void $ deleteWallet (ApiT wId)
 
+    | args `isPresent` command "transaction" && args `isPresent` command "create" = do
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Suggest wrapping:
```hs
    | args `isPresent` command "transaction" &&
      args `isPresent` command "create" = do
```

Reply:
fixed